tests/test_le.sage

```diff
@@ -0,0 +1,86 @@
+import random
+#from math import comb, comb as binomial, ceil, log2, log, factorial, sqrt, inf, floor
+from cryptographic_estimators.LEEstimator.LEAlgorithms import *
+from cryptographic_estimators.LEEstimator import LEProblem
+from cryptographic_estimators.SDFqEstimator.SDFqAlgorithms import LeeBrickell, Prange, Stern
+
+
+load('tests/module/attack_cost.sage')
+load('tests/module/cost.sage')
+
+#from module.attack_cost import *
+#from module.cost import *
+
+bbps_params = {"bit_complexities": 0, "sd_parameters": {"excluded_algorithms": [Prange, LeeBrickell]}}
+ranges = 3.
```

Comment:
this should be at most 1. I think you are forgetting to restrict the SDFq Estimator to the correct algorithms in case of beullens at least

Reply:
lets discuss this in a meeting I guess

Reply:
i got it down to 1.5 But 1 bit is impossible.